templates/CRM/Campaign/Form/Search/Common.tpl

```diff
@@ -14,10 +14,10 @@
 {if $searchVoterFor}
   {assign var='searchForm' value="search_form_$searchVoterFor"}
 {/if}
-  <div id="{$searchForm}" class="crm-accordion-wrapper crm-contribution_search_form-accordion {if $rows}collapsed{/if}">
-    <div class="crm-accordion-header {if !$votingTab} crm-master-accordion-header{/if}">
+  <details id="{$searchForm}" class="crm-accordion-wrapper crm-contribution_search_form-accordion" {if $rows}{else}open{/if}>
+    <summary class="crm-accordion-header {if !$votingTab} crm-master-accordion-header{/if}">
```

Comment:
`.crm-master-accordion-header` is the utility class that changes the look of the entire accordion from dark bg/light text with border, to no header bg and no border. It only need to be applied to .crm-accordion-header for that to work (ie not the container div/details).

But since 5.69 there's been two utility classes to trigger these two UI versions, with a goal of removing `.crm-master-accordion-header`. In this instance it would be something like: `<details class="crm-accordion-{if !$votingTab}light{else}dark{/if}">.